Classes/common/CDTIncrementalStore.m

```diff
@@ -922,11 +932,14 @@ - (BOOL)insertManagedObject:(NSManagedObject *)mo error:(NSError **)error
  *  Update existing object in the database
  *
  *  @param mo    Managed Object
+ *  @param changes Dictionary of [key:NSString, value:id] of changes to make to mo contents
  *  @param error Error
  *
  *  @return YES/NO
  */
-- (BOOL)updateManagedObject:(NSManagedObject *)mo error:(NSError **)error
+- (BOOL)updateManagedObject:(NSManagedObject *)mo
+                withChanges:(NSDictionary *)changes
```

Comment:
I don't know that I can avoid doing a fetch, because the batch update request specifies a predicate and I need to do a fetch on this predicate to know which documents to change.  Let's discuss tomorrow ...